src/database/queries/applicationForms/insertOne.sql

```diff
@@ -12,8 +12,4 @@ INSERT INTO application_forms (
     1
   )
 )
-RETURNING
-  id as "id",
-  opportunity_id as "opportunityId",
-  version as "version",
-  created_at as "createdAt"
+RETURNING application_form_to_json(application_forms) AS "object";
```

Comment:
I still ponder when I see this that every single insert causes a bunch of selects with joins. Until it becomes a performance issue it's not an issue.